### ISSUE TITLE
Fix download link on notification status screen

### ISSUE THREAD

**Issue by adriannelee** [cds-snc/notification-planning#602]
## Problem statement
[The 'Download' link is stretched to fill the width of the send status page unnecessarily. The participant questioned if there was more that he was missing to the right of the text.](https://airtable.com/appWwAmHwDLtpIyko/tbl38n7ofWYBuezFc/viwEUFgBd0KLqH7oo/recWw6qsBqHCqogQC?blocks=hide)

## Hypothesis
Fixing the download link so that the active(?) state doesn't visually stretch longer than the 'Download' text will indicate to screen magnification users that there is no more they need to read.

**Comment by amazingphilippe** [comment]
This is fixed on both `/jobs` and `/notifications` pages
[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBaXhTIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--3d867ef8af59ebd0b9aa472af3dd2de5699414db/image.png)
